tests/external_executor/external_executor.py

```diff
@@ -166,9 +172,9 @@ def test_simple_executor(network, credentials, args):
 
     with executor_thread(WikiCacherExecutor(primary, credentials)):
         with primary.client() as c:
-            c.post("/not/a/real/endpoint")
-            c.post("/update_cache/Earth")
-            c.get("/article_description/Earth")
+            c.post("/not/a/real/endpoint", timeout=2)
+            c.post("/update_cache/Earth", timeout=2)
+            c.get("/article_description/Earth", timeout=2)
```

Comment:
Not sure I understand why this timeout is added?

Reply:
Good spot! These were added as part of debugging, but are unneeded now - I've removed them.